src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,24 @@
 package racingcar;
 
+import java.util.ArrayList;
+import racingcar.domain.Car;
+import racingcar.domain.Judgement;
+import racingcar.domain.RacingProcess;
+import racingcar.domain.UserConsole;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        Judgement judgement = new Judgement();
+        UserConsole userConsole = new UserConsole();
+        RacingProcess race = new RacingProcess();
+
+        ArrayList<Car> carList = userConsole.makingCarList();
```

Comment:
```suggestion
        List<Car> carList = userConsole.makingCarList();
```
어떤 차이가 있을까요?

또, 앞에 이미 List 형이라고 명시되어 있기도 하고, 사실 자료형 자체는 언제든지 변경될 수 있기 때문에 변수를 네이밍 할 때는 자료형을 명시하지 않는 것이 일반적입니다.
그냥 'List\<Car\> cars' 로 쓸 수 있겠죠.

Reply:
그냥 생각없이 쓰고 있던 내용이었던 것 같아요. 수정했습니다.